Edited first part (#473)
mia-platform/documentation: docs/infrastructure/overview.md

Mia-Platform can be adopted in three different ways, according to your needs:

- **Platform as a Service (PaaS)**: Purchase Mia-Platform as a fully managed environment, ready to use for your development teams. Further information available [here](/paas/overview.md).
- **Bring Your Own Infrastructure (SaaS)**: Purchase Mia-Platform as a managed service but keep the runtime infrastructure under your control and management.

> - **Self Hosted (on-premises)**: Purchase Mia-Platform’s License to be installed and managed on your own infrastructure. Further information available [here](/self_hosted/self_hosted_requirements.md).

Reviewer: does it make sense to use the possessive form? I mean, it is correct but we never use it anywhere else as far as I know; i'm leaving you a suggestion here, please consider it or ignore it if it's fine using that form

```suggestion
- **Self Hosted (on-premises)**: Purchase Mia-Platform License to have everything installed and managed on your own infrastructure. Further information available [here](/self_hosted/self_hosted_requirements.md).
```

PS: I've also proposed a little rephrasing
